cli/src/commands/undo.rs

```diff
@@ -201,8 +201,10 @@ pub fn cmd_undo(ui: &mut Ui, command: &CommandHelper, args: &UndoArgs) -> Result
     );
     tx.repo_mut().set_view(new_view);
     if let Some(mut formatter) = ui.status_formatter() {
-        write!(formatter, "Restored to operation: ")?;
         let template = tx.base_workspace_helper().operation_summary_template();
+        write!(formatter, "Undid operation: ")?;
+        template.format(&op_to_undo, formatter.as_mut())?;
+        write!(formatter, "\nRestored to operation: ")?;
         template.format(&op_to_restore, formatter.as_mut())?;
```

Comment:
I think the "Restored to operation: " line can be deleted. It wouldn't be so interesting to see the parent operation of the operation to be undone.

Reply:
I decided that redo should not have a similar change. But I don’t remember my reasoning, probably that it already shows the operation you end up at?

I also think that having both lines in undo makes sense.

Reply:
> for redo it doesn't look very useful to print op_to_redo:

I think `jj redo` should print the original operation that had been undone. I didn't mean this change should be included in this PR, because we'll need non-trivial change to look up the original operation.

FWIW, in `jj undo`, "Restored to operation:" wouldn't always be the same as the next operation to undo.

Reply:
> FWIW, in jj undo, "Restored to operation:" wouldn't always be the same as the next operation to undo.

Hmm, anything we should do about that?

Are you asking for any change on this PR?

Something like "Perhaps a small window into the undo/redo stack" sounds like a good idea...

Reply:
I just mean, if the intent of leaving the "Restored to" line was to mention the next undo target, the current implementation wouldn't be good. This can be addressed later.